src/app/profile/profile.component.js

```diff
@@ -176,6 +176,12 @@ class ProfileController {
     this.pKey = this.donorDetails["acs-profile-pkey"] || this.donorDetails["acs-spouse-profile-pkey"]
   }
 
+  linkToAdobeCampaign() {
+    if (this.pKey) {
+      window.open(`https://cru-mkt-prod1-m.adobe-campaign.com/lp/LPEmailPrefCenter?_uuid=8831d67a-0d46-406b-8987-fd07c97c4ca7&_test=true&pkey=${this.pKey}&service=%400fAlW4GPmxXExp8qlx7HDlAM6FSZUd0yYRlQg6HRsO_kglfi0gs650oHPZX6LrOvg7OHoIWWpobOeGZduxdNU_m5alc`)
```

Comment:
This URL should be environment-specific, otherwise we can't test in stage.

Reply:
Should I just store the stage and prod URLs as constants in this file?  Or store them in `app.config.js`?  It's not just the domain names that are different, the values for `uuid` and `service` are also distinct between the two environments.

Reply:
Probably best to put it in `app.config.js` under the `vars`. `development` can share the `staging` value.